stable/artifactory-ha/README.md

```diff
@@ -672,7 +675,8 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.image.version`          | Container image tag                  | `.Chart.AppVersion`                        |
 | `artifactory.loggers`             | Artifactory loggers (see values.yaml for possible values) | `[]`                     |
 | `artifactory.catalinaLoggers`     | Artifactory Tomcat loggers (see values.yaml for possible values) | `[]`              |
-| `artifactory.customInitContainers`| Custom init containers                  |                                            |
+| `artifactory.customInitContainersBegin`| Custom init containers runned at the begin|                                            |
+| `artifactory.customInitContainers`| Custom init containers runned at the end                  |                                            |
```

Comment:
Rephrase description to `Custom init containers to run after existing init containers`

Reply:
Done